examples/plotting/hmi_cutout.py

```diff
@@ -0,0 +1,126 @@
+magnetogram_big = sunpy.map.Map(magnetogram.data, magnetogram.meta, mask=mask.mask)
+
+
+# The next stage is to setup the figure in two stages.
+# The first stage is dealing with the full disk image.
+
+fig = plt.figure()
+
+# We want to create a nice normalation range for the image
+
+norm = matplotlib.colors.SymLogNorm(50, vmin=-7.5e2, vmax=7.5e2)
+
+# Plot the first magnetogram
+
+ax1 = fig.add_subplot(121, projection=magnetogram_big)
+magnetogram_big.plot(axes=ax1, cmap='RdBu_r', norm=norm, annotate=False,)
+
+# These lines deal with hiding the axis, its ticks and labels
+
+lon, lat = ax1.coords[0], ax1.coords[1]
+lon.frame.set_linewidth(0)
+lat.frame.set_linewidth(0)
+lon.set_ticks_visible(False)
+lat.set_ticks_visible(False)
+lon.set_ticklabel_visible(False)
+lat.set_ticklabel_visible(False)
+
+# We will draw the rectangle around the region we plan to showcase in the cutout image.
+
+magnetogram_big.draw_rectangle(left_corner, height=right_corner.Tx-left_corner.Tx,
+                               width=right_corner.Ty-left_corner.Ty, color='k', lw=1)
+# As well as draw the full cooridate grid on top.
+
+magnetogram_big.draw_grid(axes=ax1, color='k', alpha=0.25, lw=0.5)
+
+
+# Following this, we will now deal with the zoomed-in magnetogram.
+
+magnetogram_small = magnetogram.submap(left_corner, top_right=right_corner)
+ax2 = fig.add_subplot(122, projection=magnetogram_small)
+im = magnetogram_small.plot(axes=ax2, norm=norm, cmap='RdBu_r', annotate=False,)
+ax2.grid(alpha=0)
+lon, lat = ax2.coords[0], ax2.coords[1]
+
+# Unlike the full disk image, here we will just clean up the axis labels and ticks.
+
+lon.frame.set_linewidth(1)
+lat.frame.set_linewidth(1)
+lon.set_ticklabel()
+lat.set_ticklabel(rotation='vertical',)
+lon.set_axislabel('Helioprojective Longitude',)
+lat.set_axislabel('Helioprojective Latitude',)
+lat.set_axislabel_position('r')
+lat.set_ticks_position('r')
+lat.set_ticklabel_position('r')
+
+# Now for the finishing touches, we want to add two lines that will connect
+# the two images as well as a colorbar.
+
+xpix, ypix = magnetogram_big.world_to_pixel(right_corner)
+con1 = ConnectionPatch(
+    (0, 1), (xpix.value, ypix.value), 'axes fraction', 'data', axesA=ax2, axesB=ax1,
+    arrowstyle='-', color='k', lw=1
+)
+xpix, ypix = magnetogram_big.world_to_pixel(
+    SkyCoord(right_corner.Tx, left_corner.Ty, frame=magnetogram_big.coordinate_frame))
+con2 = ConnectionPatch(
+    (0, 0), (xpix.value, ypix.value), 'axes fraction', 'data', axesA=ax2, axesB=ax1,
+    arrowstyle='-', color='k', lw=1
+)
+ax2.add_artist(con1)
+ax2.add_artist(con2)
+
+pos = ax2.get_position().get_points()
+cax = fig.add_axes([
+    pos[0, 0], pos[1, 1]+0.01, pos[1, 0]-pos[0, 0], 0.025
+])
+cbar = fig.colorbar(im, cax=cax, orientation='horizontal', label="abc")
+
+# For the colorbar we want it to have three fixed ticks
+
+cbar.locator = matplotlib.ticker.FixedLocator([-1e2, 0, 1e2])
+cbar.set_label("LOS Magnetic Field [Gauss]", labelpad=-40, rotation=0)
```

Comment:
```suggestion
cbar.set_label("LOS Magnetic Field [gauss]", labelpad=-40, rotation=0)
```
The unit "gauss" is not capitalized.